.gitignore

```diff
@@ -1,11 +1,12 @@
 /bazel-*
 /build_*
 /dist/envoy.aar
-/dist/Envoy.framework
 .DS_Store
 /generated
 .idea
 .ijwb
 *.pyc
 tags
 .vscode
+
+*/**/Envoy.framework
```

Comment:
Writing to our workspace inside of dist/ as part of the build is already a little bit iffy - is it possible to avoid doing it elsewhere (e.g. by building the app with bazel)?
OTOH if we're not planning on merging this, it's not a big deal.

Reply:
This PR isn't being merged, it's here purely for ease of pushing test code somewhere that's visible from linked issues

Reply:
🚀